homeassistant/components/volvooncall/device_tracker.py

```diff
@@ -23,8 +23,9 @@ async def async_setup_scanner(
         return False
 
     vin, component, attr, slug_attr = discovery_info
-    data = hass.data[DATA_KEY]
-    instrument = data.instrument(vin, component, attr, slug_attr)
+    coordinator: VolvoUpdateCoordinator = hass.data[DATA_KEY]
+    volvo_data = coordinator.volvo_data
+    instrument = volvo_data.instrument(vin, component, attr, slug_attr)
```

Comment:
The code below will fail if `instrument` is `None`. Can that happen? Should we update the code here or in volvo_data?

Reply:
added checks for None, and some logging in volvo_data should that occur